adapter/internal/adapter/adapter.go

```diff
@@ -312,4 +312,7 @@ func fetchAPIsOnStartUp(conf *config.Config) {
 			}(data)
 		}
 	}
+	// All apis are fected. Deploy the /ready route for the readiness and startup probes.
```

Comment:
```suggestion
	// All apis are fetched. Deploy the /ready route for the readiness and startup probes.
```

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2101/commits/f8b1a0f26d76982947e760b92ce30641bbf7e936